### PR TITLE
[ci][api] Enable Performance/StringReplacement rubocop cop

### PR DESCRIPTION
**Pull request by DavidKang**

### CODE REVIEW

File: src/api/.rubocop.yml [hunk 1]
```diff
@@ -148,6 +140,21 @@ Style/TernaryParentheses:
 # Checks for predicates than can be replaced by receiver.empty? and !receiver.empty?
 Style/ZeroLengthPredicate:
   Enabled: true
+
+##################### Performance ###############################
```
Comment (Ana06) [inline src/api/.rubocop.yml]: Performance in the Rubocop default config file is between Lint and Rails. Not sure why but maybe our config file should be ordered in the same way. :thinking:

File: src/api/app/controllers/webui/search_controller.rb [hunk 2]
```diff
@@ -108,7 +108,7 @@ def set_parameters
 
     @search_text = ""
     @search_text = params[:search_text].strip unless params[:search_text].blank?
-    @search_text = @search_text.gsub("'", '').gsub('[', '').gsub(']', '').gsub("\n", '')
+    @search_text = @search_text.delete("'").delete('[').delete(']').delete("\n")
```
Comment (bgeuken) [inline src/api/app/controllers/webui/search_controller.rb]: JFYI, this could also be done by providing an array.

**Comment by bgeuken** [comment]
LGTM

File: src/api/.rubocop.yml [hunk 3]
```diff
@@ -199,6 +192,21 @@ Lint/UselessAccessModifier:
...
+
+# Identifies places where Hash#merge! can be replaced by Hash#[]=
+Performance/RedundantMerge:
+  Enabled: true
+
+# Identifies places where gsub can be replaced by tr or delete.
+Performance/StringReplacement:
+  Enabled: true
+
+
```
Comment (Ana06) [inline src/api/.rubocop.yml]: Here there is one more empty line than everywhere else, right? :confused:

**Pull request merged**